network/p2p/tracer/gossipSubMeshTracer.go

```diff
@@ -450,6 +479,22 @@ func (t *GossipSubMeshTracer) LastHighestIHaveRPCSize() int64 {
 	return t.rpcSentTracker.LastHighestIHaveRPCSize()
 }
 
+// DuplicateMessageCount returns the current duplicate message count for the peer.
+func (t *GossipSubMeshTracer) DuplicateMessageCount(peerID peer.ID) float64 {
+	count, found, err := t.duplicateMessageTrackerCache.GetWithInit(peerID)
+	if err != nil {
+		t.logger.Err(err).
+			Bool(logging.KeyNetworkingSecurity, true).
+			Str("peer_id", peerID.String()).
```

Comment:
```suggestion
			Str("peer_id", p2plogging.PeerId(peerID)).
```
https://github.com/onflow/flow-go/pull/5058#discussion_r1435337437